README.md

```diff
@@ -4,30 +4,135 @@
 
 The **CLIP** model originally was proposed in [Learning Transferable Visual Models From Natural Language Supervision](https://cdn.openai.com/papers/Learning_Transferable_Visual_Models_From_Natural_Language_Supervision.pdf).
 
-`ClipImageEncoder` encode images stored in the blob attribute of the **Document** and saves the encoding in the embedding attribute.
+`ClipImageEncoder` encode images stored in the blob attribute of the [**Document**](https://github.com/jina-ai/jina/blob/master/.github/2.0/cookbooks/Document.md) and saves the encoding in the embedding attribute.
 
-- Input shape: ndarray `Height x Width x Color Channel`, min=0, max=1. If `use_default_preprocessing` is `true`, input images can have any height and width. Otherwise, the input format has to be 224x224x3.
 
-- Output shape: `EmbeddingDimension`
 
-      
+## Prerequisite
 
-## Example:
+Document, Executor, and Flow are the three fundamental concepts in Jina.
 
-Here is an example usage of the clip encoder.
+- [**Document**](https://github.com/jina-ai/jina/blob/master/.github/2.0/cookbooks/Document.md) is the basic data type in Jina;
+- [**Executor**](https://github.com/jina-ai/jina/blob/master/.github/2.0/cookbooks/Executor.md) is how Jina processes Documents;
+- [**Flow**](https://github.com/jina-ai/jina/blob/master/.github/2.0/cookbooks/Flow.md) is how Jina streamlines and scales Executors.
 
-```python
-    def process_response(resp):
-        ...
+*Learn them all, nothing more, you are good to go.*
+
+
+## Usage
+
+### Via Jinahub
+
+1. Clone the repo and build the Jinahub image
+
+	```shell
+	git clone https://github.com/jina-ai/executor-clip-image.git
+	cd executor-clip-image
+	jina hub push .
+	```
+
+1. Use the image in your codes
+
+	```python
+	from jinahub.encoder.clip_image import ClipImageEncoder
+	from jina import Flow, Document
+	import numpy as np
+	
+	f = Flow().add(
+	        uses=jinahub+docker://xgdfljc:v1,
+	        volumes='/your_home_folder/.cache/clip:/root/.cache/clip')
+	
+	def check_emb(resp):
+	    for doc in resp.data.docs:
```

Comment:
I think for this u need to access `resp.docs`, otherwise won't work

Reply:
Nope, it works.